Remove parallel usage in setup and test commands

mitchellhenke opened the pull request:
## 🛠 Summary of changes

Based on discussion at yesterday's Engineering Huddle and people running into issues with the parallel testing being used by default, this PR removes `parallel_tests` usage from the setup and test commands. It is still available for those who use it, but it will have to be run directly. I've also updated the gem.

<!--
## 📜 Testing Plan

Provide a checklist of steps to confirm the changes.

- [ ] Step 1
- [ ] Step 2
- [ ] Step 3
-->

<!--
## 👀 Screenshots

If relevant, include a screenshot or screen capture of the changes.

<details>
<summary>Before:</summary>

</details>

<details>
<summary>After:</summary>

</details>
-->

aduth commented:
>It is still available for those who use it, but it will have to be run directly. 

Should we just drop it? YAGNI and such, and whether we can trust that it actually works.

mitchellhenke commented:
> > It is still available for those who use it, but it will have to be run directly.
> 
> Should we just drop it? YAGNI and such, and whether we can trust that it actually works.

Sounds good to me, dropped it.

> Another reference to remove?
> 
> https://github.com/18F/identity-idp/blob/ef0c06ac8d0d1046d46630e9706798daff05e42e/bin/docker_setup#L66-L67

Oops, thank you, removed that as well.
